src/lib/features/productivity-report/productivity-report-view-model.ts

```diff
@@ -23,69 +26,82 @@ export const productivityReportViewModel = ({
-        const healthColor =
-            healthRating >= 0 && healthRating <= 24
-                ? RED
-                : healthRating >= 25 && healthRating <= 74
-                  ? ORANGE
-                  : GREEN;
-        return healthColor;
-    },
-    actionText(): string | null {
-        const improveMessage =
-            'Remember to archive stale flags to reduce technical debt and keep your project healthy';
-        const previousHealth = this.previousMonth?.health || 0;
-        if (this.health <= 74) {
-            return improveMessage;
-        }
-        if (this.health < previousHealth) {
-            return improveMessage;
-        }
-        return null;
-    },
-    healthTrendMessage() {
-        return this.previousMonthText(
-            '%',
-            this.health,
-            this.previousMonth?.health,
-        );
-    },
-    flagsCreatedTrendMessage() {
-        return this.previousMonthText(
-            '',
-            this.flagsCreated,
-            this.previousMonth?.flagsCreated,
-        );
-    },
-    productionUpdatedTrendMessage() {
-        return this.previousMonthText(
-            '',
-            this.productionUpdates,
-            this.previousMonth?.productionUpdates,
-        );
-    },
-    previousMonthText(
-        unit: '' | '%',
-        currentValue: number,
-        previousValue?: number,
-    ) {
-        if (previousValue == null) {
+}) => {
+    const { health, previousMonth, flagsCreated, productionUpdates } = metrics;
+    const technicalDebt = Math.max(0, 100 - health) || 0;
+
+    return {
+        userName,
+        userEmail,
+        flagsCreated: flagsCreated,
+        productionUpdates: productionUpdates,
+        previousMonth,
+        health: health,
+        technicalDebt: technicalDebt.toString(),
+        unleashUrl,
+        technicalDebtColor() {
+            if (technicalDebt < 25) {
+                return GREEN;
+            }
+            if (technicalDebt < 75) {
+                return ORANGE;
+            }
+            return RED;
+        },
+        actionText(): string | null {
+            const improveMessage =
+                'Remember to archive stale flags to reduce technical debt and keep your project healthy';
+            const previousHealth = previousMonth?.health || 0;
+            if (health <= 74) {
+                return improveMessage;
+            }
+            if (health < previousHealth) {
+                return improveMessage;
+            }
             return null;
-        }
-        if (currentValue > previousValue) {
-            return `<span style='color: ${GREEN}'>&#9650;</span> ${currentValue - previousValue}${unit} more than previous month`;
-        }
-        if (previousValue > currentValue) {
-            return `<span style='color: ${RED}'>&#9660;</span> ${previousValue - currentValue}${unit} less than previous month`;
-        }
-        return `Same as last month`;
-    },
-});
```

Comment:
`previousMonthText` abstraction is not needed - too many exceptions from the rule